coursebuilder/modules/explorer/_static/css/style.css

```diff
@@ -3594,7 +3594,7 @@ footer .footer-bottom .footer-social-icons a { color: black; }
 
 .course-card { position: relative; }
 
-.course-card img { width: 100%; }
+.course-card img { width: 100%; max-height: 400px; }
```

Comment:
@andela-iakande max-width: 100% and not width: 100%

Reply:
@chege done.